modules/postgresql/gcp/variables.tf

```diff
@@ -45,8 +45,15 @@ variable "provision_read_replica" {
 variable "big_query_connection_location" {
   default = "US"
 }
-variable "upgradable" {
-  description = "Instance is upgradable via Database Migration Service"
+
+variable "source_db_upgradable" {
```

Comment:
Something is weird with the `_upgradeable` postfix. how about:

`prep_upgrade_as_source_db`?